oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/register/ServiceInstanceInventory.java

```diff
@@ -199,6 +199,7 @@ public boolean hasProperties() {
         public static final String PROCESS_NO = "process_no";
         public static final String IPV4S = "ipv4s";
         public static final String LANGUAGE = "language";
+        public static final String DATABASE = "database";
```

Comment:
Why add DATABASE here? Database just generate network and service, no service instance, I think. Right?